py/tests/test_subscriptions.py

```diff
@@ -0,0 +1,15 @@
+from tests.test_case import *
+from app.pcasts.dao import subscriptions_dao # pylint: disable=C0413
+
+class SubscriptionsTestCase(TestCase):
+
+  def setUp(self):
+    super(SubscriptionsTestCase, self).setUp()
+    Subscription.query.delete() # delete all existing subscriptions
+    db_session_commit()
+
+  def test_get_subscriptions(self):
+    assert not subscriptions_dao.get_user_subscriptions(1)
```

Comment:
`unittest.TestCase` has convenience methods for assertion like `self.assertFalse`, along with `assertEquals`, `assertRaises` and others

Reply:
Ah I see, IMO it's not necessary to change for this PR, just wanted to write some dummy test to make sure things are working, I'll change it when I actually write the tests for the subscriptions